src/dataset/impl_dataset.rs

```diff
@@ -288,6 +288,16 @@ impl<F: Float, T: Targets, D: Data<Elem = F>, I: Dimension> From<(ArrayBase<D, I
 }
 
 impl<F: Float, E: Copy> Dataset<F, E> {
+    /// Bootstrap datasets by samples sampling with replacement
```

Comment:
I find this sentence a little confusing. Maybe it should be "by sampling samples"? I'm not a native speaker so I might be wrong 😅. I really like the wording on the [Wikipedia article](https://en.wikipedia.org/wiki/Bootstrap_aggregating#Description_of_the_technique), something along the lines of "Applies bootstrapping (to generate new datasets) by sampling uniformly and with replacement from the original dataset", where maybe the part between the parentheses could be excluded